core/src/main/scala/org/apache/spark/sql/execution/ddl.scala

```diff
@@ -628,6 +626,40 @@ case class ListPackageJarsCommand(isJar: Boolean) extends RunnableCommand {
 case class UnDeployCommand(alias: String) extends RunnableCommand {
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
+    var value = ""
+    val sc = sparkSession.sparkContext
+    if (alias != null) {
+      val cmndsSet = ToolsCallbackInit.toolsCallback.getGlobalCmndsSet
+      cmndsSet.forEach(new Consumer[Entry[String, String]] {
+        override def accept(t: Entry[String, String]): Unit = {
+          val alias1 = t.getKey
+          if(alias == alias1) {
+            value = t.getValue
+          }
+        }
+      })
+      val indexOf = value.indexOf("|")
```

Comment:
Please see if you can re-use the code from `SnappyContext.initGlobalSnappyContext()` and avoid code duplication.

Reply:
Will try to remove this duplicate code in later commits.